composer.json

```diff
@@ -11,7 +11,7 @@
 		}
 	],
 	"require": {
-		"php": ">=5.6",
+		"php": ">=7.4",
```

Comment:
Even if this change was deemed desirable not (I don't think it is), then I'd also expect changes to readme, CI, and Requires PHP plugin header.